dev/src/lobster/glinterface.h

```diff
@@ -90,6 +92,22 @@ struct Shader : lobster::Resource {
     }
 };
 
+struct TimeQuery : lobster::Resource {
+    bool active = false;
```

Comment:
tip: a constant only needed inside a class can be nicely defined inside the class with an `enum { TIME_QUERY_SAMPLE_COUNT = 8 };`